caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.columns.js

```diff
@@ -1226,17 +1227,31 @@
     if ($output.text())
     {
       var currentURI = cadc.web.util.currentURI();
+      // Need to strip off tab # anchor tag...
+
 
       $.each(_searchItems, function (name, value) {
-        currentURI.setQueryValue(name, encodeURIComponent(value));
+        if (name != "useMaq") {
+          currentURI.setQueryValue(name, encodeURIComponent(value));
+        }
       });
 
       // Then issue the href (target) of the link.
       var $link = $("<a></a>");
 
       // Force open in a new window.  This MUST be set to _blank.
       $link.attr("target", "_blank");
-      $link.attr("href", currentURI.toString());
+
+      var useMaqStr = "&useMaq=";
```

Comment:
This could just be `var useMaqStr = "&useMaq=" + !(($("#resultsMaqEnabled") == "undefined") || $("#resultsMaqEnabled").hasClass("cadc-display-none"))`.